SessionMessagingKit/Jobs/Types/DisappearingMessagesJob.swift

```diff
@@ -54,7 +55,7 @@ public extension DisappearingMessagesJob {
         // If there is another expiring message then update the job to run 1 second after it's meant to expire
         let nextExpirationTimestampMs: Double? = try? Interaction
             .filter(Interaction.Columns.expiresStartedAtMs != nil)
-            .filter(Interaction.Columns.expiresInSeconds != nil)
+            .filter(Interaction.Columns.expiresInSeconds > 0)
```

Comment:
🤔 The `expiresInSeconds` shouldn't be negative, it should be a positive number and only from the options we provide, but to be safe, I'll make the filter to be `!= 0` since `0` means disappearing messages off.